static/js/hours/table/table-strings-i18n.js

```diff
@@ -28,6 +29,56 @@ export default function provideTableHeadersTranslation(locale) {
         [TableHeaders.DAY_OF_WEEK]: '曜日',
         [TableHeaders.HOURS]: '営業時間'
       }
+    case 'ar':
+      return {
+        [TableHeaders.DAY_OF_WEEK]: 'يوم من الأسبوع',
+        [TableHeaders.HOURS]: 'ساعات'
+      }
+    case 'hi':
+      return {
+        [TableHeaders.DAY_OF_WEEK]: 'सप्ताह का दिन',
+        [TableHeaders.HOURS]: 'घंटे'
+      }
+    case 'ko':
+      return {
+        [TableHeaders.DAY_OF_WEEK]: '요일',
+        [TableHeaders.HOURS]: '시간'
+      }
+    case 'nl':
+      return {
+        [TableHeaders.DAY_OF_WEEK]: 'dag van de week',
+        [TableHeaders.HOURS]: 'uur'
+      }
+    case 'pl':
+      return {
+        [TableHeaders.DAY_OF_WEEK]: 'Dzień tygodnia',
+        [TableHeaders.HOURS]: 'godziny'
+      }
+    case 'pt':
+      return {
+        [TableHeaders.DAY_OF_WEEK]: 'Dia da semana',
+        [TableHeaders.HOURS]: 'Horas'
+      }
+    case 'ru':
+      return {
+        [TableHeaders.DAY_OF_WEEK]: 'День недели',
+        [TableHeaders.HOURS]: 'Часы'
+      }
+    case 'sv':
+      return {
+        [TableHeaders.DAY_OF_WEEK]: 'Veckodag',
+        [TableHeaders.HOURS]: 'Timmar'
+      }
+    case 'zh-CH':
```

Comment:
We're actually in the process of cutting over to 'zh-Hans' and 'zh-Hant'. Turns out, that's what LiveAPI and KG use instead. We want to be consistent with the rest of Yext. You should just be able to update the key names here.